ert/data/record/_transmitter.py

```diff
@@ -78,13 +96,64 @@ async def _load_numerical_record(self) -> NumericalRecord:
     async def _load_blob_record(self) -> BlobRecord:
         raise NotImplementedError("not implemented")
 
+    @abstractmethod
+    async def get_recordtree_transmitters(
```

Comment:
should this function be private?

Reply:
I thought about it, but it might still be used in transformations, but as it is not in place yet I guess it is fine to make it private.